light/provider/http/http.go

```diff
@@ -20,21 +20,46 @@ import (
 var defaultOptions = Options{
 	MaxRetryAttempts: 5,
 	Timeout:          3 * time.Second,
+  NoBlockThreshold:    5,
```

Comment:
formatting seems to be off

Reply:
weird, my editor must not be picking it up. Thanks for pointing it out